README.md

```diff
@@ -1,24 +1,597 @@
-# README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<a name="readme-top"></a>
+<!-- PROJECT SHIELDS -->
 
-Things you may want to cover:
+[![Tavern Keeper](https://img.shields.io/badge/Tavern_Keeper-Visit_Website-purple?style=for-the-badge)](INSERT-DEPLOY-LINK)
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Issues][issues-shield]][issues-url]
 
-* Ruby version
 
-* System dependencies
 
-* Configuration
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <a href="https://github.com/Tavern-Keeper-2308/tavern_keeper_be">
+    <img src="images/logo.png" alt="Logo" width="80" height="80">
+  </a>
 
-* Database creation
+<h3 align="center">Tavern Keeper</h3>
 
-* Database initialization
+  <p align="center">
+    INSERT-DETAILS-HERE
+    <br />
+    <a href="https://github.com/Tavern-Keeper-2308/tavern_keeper_be"><strong>Explore the docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://github.com/Tavern-Keeper-2308/tavern_keeper_be">View Demo</a>
+    ·
+    <a href="https://github.com/Tavern-Keeper-2308/tavern_keeper_be/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/Tavern-Keeper-2308/tavern_keeper_be/issues">Request Feature</a>
+  </p>
+</div>
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
 
-* Deployment instructions
+<!-- TABLE OF CONTENTS -->
+## Table of Contents
+  1. [About The Project](#about-the-project)
+      - [Built With](#built-with)
+      - [Gems](#gems)
+  2. [Getting Started](#getting-started)
+      - [Installation](#installation)
+      - [Testing](#testing)
+  3. [Database](#database)
+  4. [Endpoints](#endpoints)
```

Comment:
I think you're right and these links aren't working currently.

Reply:
I just pushed a recent commit that fixed these.
The anchor structures I will use moving on forward are `q-something` or `m-something-something`